newsroom/core/resources/validators.py

```diff
@@ -116,3 +118,39 @@ async def _validate_iunique_value(item: ResourceModel, value: Any) -> None:
             )
 
     return AsyncValidator(_validate_iunique_value)
+
+
+def validate_valid_objectid(field_name: str) -> BeforeValidator:
```

Comment:
@eos87 This should not be needed as we already have [superdesk.core.resources.fields.ObjectId](https://github.com/superdesk/superdesk-core/blob/async/superdesk/core/resources/fields.py#L144) that'll do this validation for us

Reply:
Good one, I'll adjust it.